packages/core/story-structure.js

```diff
@@ -42,6 +42,7 @@ const s = [
           { n: 'Tearsheet', s: ['c/Tearsheet', 'c/TearsheetNarrow'] },
           { n: 'DataSpreadsheet', s: ['c/DataSpreadsheet'] },
           { n: 'Datagrid', s: ['c/Datagrid'] },
+          { n: 'ComboButton', s: ['c/ComboButton']}
```

Comment:
Our order looks loosely alphabetical ... Maybe we should move this after `Card`?

Reply:
Yes, I thought alphabetical would be best but it wasn't presented that way so I figured it was to be added at the bottom. I can make the list alphabetical if that's the way we want to go.